src/hotspot/share/compiler/disassembler.hpp

```diff
@@ -25,15 +25,15 @@
 #ifndef SHARE_COMPILER_DISASSEMBLER_HPP
 #define SHARE_COMPILER_DISASSEMBLER_HPP
 
-#include "utilities/globalDefinitions.hpp"
-
 #include "asm/assembler.hpp"
 #include "code/codeBlob.hpp"
 #include "code/nmethod.hpp"
 #include "compiler/abstractDisassembler.hpp"
 #include "runtime/globals.hpp"
+#include "utilities/globalDefinitions.hpp"
 #include "utilities/macros.hpp"
 
+
```

Comment:
Extra blank line inserted?

Reply:
This seems to be left-overs from an earlier run. If I run the tool on this file it doesn't add this blank line.